packages/shared/src/models/IApplicationState.ts

```diff
@@ -18,11 +18,13 @@
 
 import type { PodInfo } from '@podman-desktop/api';
 
+export type PodHealth = '' | 'starting' | 'healthy' | 'unhealthy';
```

Comment:
Why are you using '' it is not a valid value ? if we want to know when we do not have this value, we should use health?: PodHealth bellow

Reply:
the empty string is the value returned by Inspect when a container has no healthcheck configured (or 'none' as documented in https://docs.docker.com/engine/api/v1.45/#tag/Container/operation/ContainerInspect)

My idea is to have the same value as for a container

Reply:
I understand, as much as I hate to see there api to provide that value, your implementation make sense with it thanks for pointing this out

Reply:
I can replace the empty string with the `'none'` one to completely stick with the documented values

Reply:
I also prefer none because this seems to be also what podman return https://github.com/containers/podman/blob/8ec821452581120b73625daff68b5eb4829f44f6/pkg/api/server/register_containers.go#L79